fplll/pruner/pruner.h

```diff
@@ -128,6 +192,14 @@ void prune(/*(input)output*/ PruningParams &pruning,
            const PrunerMetric metric = PRUNER_METRIC_PROBABILITY_OF_SHORTEST,
            const int flags           = PRUNER_GRADIENT);
 
+template <class FT>
+double prune_cost(/*(input)output*/ PruningParams &pruning,
```

Comment:
This function doesn't have documentation. Also, AFAIK `pruning` is not an output for this function?

Reply:
My bad. This function is not used anywhere. I will remote it.